### PR TITLE
Ensure a student's current team is from the current season

### PR DESCRIPTION
**Pull request by carpodaster**

### CODE REVIEW

**Comment by klappradla** [comment]
Since this is lying around for some time now 💤  - what's the state of this? Is this WIP or should this be reviewed and merged soonish @carpodaster ? Or is this no longer relevant?

**Comment by carpodaster** [comment]
@klappradla totally forgot about this one. No, it's not WIP, but I wish I had used a PR template and linked a related bug :wink:  I would think it is related to #621 ? :thinking:

**Comment by mkalininait** [comment]
@carpodaster I think you pushed this PR while fixing this bug: https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/803
cc @klappradla

File: spec/models/student_spec.rb [hunk 1]
```diff
@@ -32,14 +32,52 @@
...
+        end
+      end
+
+      context 'when the user has been a student before but is now contributing to the project' do
+        before do
+          create :student_role, user: user, team: old_team
+          create :supervisor_role, user: user, team: current_team
+        end
+
+        it 'returns the current season\'s team' do
```
Comment (klappradla) [inline spec/models/student_spec.rb]: I guess this should be: `it 'does not return any team'` 🤔

**Comment by carpodaster** [comment]
Thank you @klappradla – fixed!

**Comment by klappradla** [comment]
👍 waiting for CI, will then do mergefest

**Pull request merged**